docs/element-definitions.md

```diff
@@ -86,10 +86,42 @@ The value of the `Templated Href` type SHOULD be resolved as a URI-Reference per
 
 The definition is a Data Structure element `Object Type` where keys are respective URI Template variables.
 
+**NOTE:** *A href variable may include an attribute `type` which may
+contain a label for the variable type.*
+
 #### Properties
 
 - `element`: hrefVariables (string, fixed)
```

Comment:
You're exactly right! I wonder if there needs to be a special kind of `member` created or something like you have.